components/branches/CreateBranchModal.tsx

```diff
@@ -26,9 +26,14 @@ interface Project {
 interface BranchModalProps {
   isOpen: boolean;
   setIsOpen: Dispatch<SetStateAction<boolean>>;
+  onSuccessCreation: () => void;
 }
 
-const CreateBranchModal = ({ isOpen, setIsOpen }: BranchModalProps) => {
+const CreateBranchModal = ({
+  isOpen,
+  setIsOpen,
+  onSuccessCreation,
```

Comment:
Lets rename to `onSuccess` , const will self define that you are creating something.

Reply:
![image](https://user-images.githubusercontent.com/3684236/222378860-f883a971-f3c4-4a6d-9c87-494654a57e3b.png)